### ISSUE TITLE
quality option deprecation warning

### ISSUE THREAD

**Issue by Munter** [papandreou/express-processimage#23]
When running assetgraph-builders `processImages` transform I'm getting this console output:

```
(node:49199) DeprecationWarning: quality: use jpeg({ quality: ... }), webp({ quality: ... }) and/or tiff({ quality: ... }) instead
(node:49199) DeprecationWarning: progressive: use jpeg({ progressive: ... }) and/or png({ progressive: ... }) instead
```

I have a few images that use progressive scan jpegs and set the quality: `images/testimonial-1.png?extract=0,0,720,400&jpeg&quality=70&progressive`

**Comment by papandreou** [comment]
Fixed in 7.2.1.

**Comment by Munter** [comment]
Awesome!
